app/renderer/components/styles/global.js

```diff
@@ -106,6 +106,7 @@ const globalStyles = {
     navbarBraveButtonMarginLeft: '80px',
     navbarLeftMarginDarwin: '76px',
     sideBarWidth: '190px',
+    aboutPageDetailsPageWidth: '704px',
```

Comment:
I think I named so in order to replace other ["704px"s](https://github.com/brave/browser-laptop/search?utf8=%E2%9C%93&q=704px) on about pages with that variable.

Is it OK to do so with this PR or another one?